mongodbforms/documents.py

```diff
@@ -357,7 +358,7 @@ def __new__(cls, name, bases, attrs):
         return new_class
 
 
-class BaseDocumentForm(BaseForm):
+class BaseDocumentForm(BaseModelForm):
```

Comment:
Absolutely! (and if it doesn't, it will be thoroughly tested over time in devops-connector – which is part of why I want to make these changes sooner rather than later so that the non critical devcon system can break if need be).